src/Streamly/Internal/Data/Fold.hs

```diff
@@ -1687,15 +1861,15 @@ zipWithM = undefined
 
 -- | Zip a stream with the input of a fold.
 --
--- /Unimplemented/
+-- /Pre-release/
```

Comment:
zipWithM is undefined, so this should remain Unimplemented. Same for `indexed` and `sampleFromThen`.